example/serialized_request/web-client/index.html

```diff
@@ -0,0 +1,162 @@
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <script src="https://cdn.jsdelivr.net/npm/axios/dist/axios.min.js"></script>
+  <link href="style.css" rel="stylesheet" type="text/css">
+  <script
+    src="scalardl-web-client-sdk.bundle.js"></script>
+</head>
+
+<body>
+<div id="main-container">
+  <img src="../img/scalar_logo_horizontal.jpg" width="25%">
+  <h2 style="color: rgb(60, 103, 177)">Serialized request execution sample</h2>
+  <div class="Component">
+    <button id="registerCertificateBtn">
+      Register Certificate
+    </button>
+    <pre><code>
+await clientService.createSerializedCertificateRegistrationRequest();</code></pre>
+    <br>
+  </div>
+  <pre><code id="registerCertificateResult"></code></pre>
+  <br>
+  <br>
+  <div style="color: rgb(90, 90, 90)">
+    Upload web-client/StateUpdater.class contract: <input id="contractBinary" type="file"/>
+  </div>
+  <div class="Component">
+    <button id="registerContractsBtn">
+      Register Contracts
+    </button>
+    <pre><code>
+await clientService.createSerializedContractRegistrationRequest(
+            contractId, 'com.org1.contract.StateUpdater', contractBinary, {'properties': ''});</code></pre>
+  </div>
+  <pre><code id="registerContractResult"></code></pre>
+  <br>
+
+  <div class="Component">
+    <button id="listContractsBtn">
+      List Contracts
+    </button>
+    <pre><code>
+ await clientService.createSerializedContractsListingRequest();</code></pre>
+  </div>
+  <pre><code id="listContractsResult"></code></pre>
+</div>
+
+</body>
+
+<script>
+  // ------------- Web SDK Code -------------
+  let firstItem;
+  let contractBinary;
+  let jsonValue;
+  const webClientAddress = 'http://localhost:3002/';
+  const clientProperties = {
+    //TODO Remove this 3 properties when this issue is resolved https://github.com/scalar-labs/scalardl-web-client-sdk/issues/33
+    'scalar.dl.client.server.host': '127.0.0.1',
+    'scalar.dl.client.server.port': 80,
+    'scalar.dl.client.server.privileged_port': 8080,
```

Comment:
Before we merge this PR, I suppose it's better to to wait for https://github.com/scalar-labs/scalardl-web-client-sdk/issues/33 to be fixed.

Reply:
These 3 properties were removed in https://github.com/scalar-labs/scalardl-node-client-sdk/pull/18/commits/b8665a35b63d0eeab95fb90b23d7576c651eb59a